rocky/assets/js/renameReports.js

```diff
@@ -1,35 +1,38 @@
 import { onDomReady } from "../js/imports/utils.js";
 
-onDomReady(initConfirmButtons);
+onDomReady(initResetButtons);
 
-export function initConfirmButtons(element) {
-  let root = element || document;
-  let confirm_buttons = root.querySelectorAll(".confirm-button");
+function initResetButtons() {
+  const resetButtons = document.querySelectorAll(".reset-button");
 
-  confirm_buttons.forEach((button) => initClickHandlers(button));
-}
+  resetButtons.forEach((button) => {
+    button.classList.add("hidden");
+
+    let input = button.closest("tr").querySelector(".name-input");
 
-function initClickHandlers(button) {
-  button.addEventListener("click", function (event) {
-    const target = event.target.closest("dialog");
+    input.defaultvalue = input.value;
 
-    editReportName(target);
+    button.addEventListener("click", function (event) {
+      input.value = input.defaultvalue;
+      button.classList.add("hidden");
+    });
   });
+
+  watchInputChanges();
```

Comment:
You could consider using event delegation instead of adding event listeners to each individual button and input. E.g.

```js
function initRenameReports() {
  const table = document.querySelector(".reset-button").closest("table");
  if (!table) return;

  table.addEventListener("click", function (event) {
    if (!event.target.matches(".reset-button")) return;
    event.stopPropagation();
    const button = event.target;
    const input = button.closest("tr")?.querySelector(".name-input");
    if (!input) return;
    input.value = input.defaultvalue;
    button.classList.add("hidden");
  });

  table.addEventListener("change", function (event) {
    if (!event.target.matches(".name-input")) return;
    event.stopPropagation();
    const input = event.target
    const button = input.closest("tr")?.querySelector(".reset-button");
    if (!button) return;
    if (input.defaultValue === input.value) {
      button.classList.add("hidden");
    } else {
      button.classList.remove("hidden");
    }
  });
}
```